deploy/modules/website/dns_record.tf

```diff
@@ -1,7 +1,13 @@
 resource "aws_route53_record" "website_record" {
   zone_id = data.aws_route53_zone.zone.id
   name    = var.env == "prod" ? var.route53_zone_name : "${var.env}.${var.route53_zone_name}"
-  type    = "CNAME"
-  ttl     = "300"
-  records = [aws_cloudfront_distribution.website_distribution.domain_name]
+  type    = "A"
+  //ttl     = "300"
+  //records = [aws_cloudfront_distribution.website_distribution.domain_name]
+
+  alias {
+    name                   = aws_cloudfront_distribution.website_distribution.domain_name
+    zone_id                = aws_cloudfront_distribution.website_distribution.hosted_zone_id
+    evaluate_target_health = true
```

Comment:
This makes me think, maybe we should talk about HA of this app some day :d

Reply:
Yeah we can, I do not see many things… maybe duplicate the bucket in order to have two distinct  AZ for the static hosting. Lambdas are already HA.